Cargo.toml

```diff
@@ -1,29 +1,33 @@
 [package]
 name = "mediawiki"
 description = "A MediaWiki client library"
-keywords = ["MediaWiki","API"]
-categories = ["api-bindings","authentication"]
+keywords = ["MediaWiki", "API"]
+categories = ["api-bindings", "authentication"]
```

Comment:
This isn't strictly necessary, but I don't see any harm in doing it either.

Reply:
That was the formatter of the [Even better toml](https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml) plugin.